docs/compliance/index.md

```diff
@@ -4,10 +4,27 @@ keywords: standards, compliance, security
 title: Docker standards and compliance
 ---
 
-We want to ensure that Docker Enterprise Edition can be used in a manner that meets the requirements of various security and compliance standards. You can use this documentation in order to validate your Docker Enterprise Edition deployment against applicable security controls and configuration baselines. The documents that we've highlighted thus far are as follows:
+Docker Enterprise Edition can be configured and used in a manner that can help your organization satisfy the requirements of various security and compliance standards. You can use this documentation in order to validate your Docker Enterprise Edition deployment against applicable security controls and configuration baselines. The catalogs, frameworks, publications and benchmarks that we've highlighted thus far are as follows:
+
+Catalogs:
 
 - [NIST Special Publication (SP) 800-53 Revision 4](nist/800_53.md)
+
+Frameworks:
+
 - [Federal Risk and Authorization Management Program (FedRAMP)](fedramp.md)
-- [CIS Docker Benchmark](cis.md)
 
-Docker also maintains an open source repository at [https://github.com/docker/compliance](https://github.com/docker/compliance) where you can find a number of compliance resources. These include tools for auto-generating security documentation, auditing Docker Enterprise Edition systems against the security controls and even an experimental natural language processing utility for proofreading security narratives.
+Publications:
+
+- [NIST Special Publication (SP) 800-190](nist/800_190.md) - Application Container Security Guide
+- [NIST Interagency Report (NISTIR) 8176](nist/nistir_8176.md) - Security Assurance Requirements for Linux Application Container Deployments
+- [NIST Information Technology Laboratory (ITL) Bulletin October 2017](nist/itl_october2017.md) - NIST Guidance on Application Container Security
+
+Benchmarks:
+
+- [CIS Docker Benchmark](cis/docker_ce.md)
+- [CIS Kubernetes Benchmark](cis/k8s.md)
+
+Docker also maintains an open source repository at [https://github.com/docker/compliance](https://github.com/docker/compliance) where you can find a number of machine-readble compliance resources in addition to the source of this documentation. This also includes tools for auto-generating security documentation, auditing Docker Enterprise Edition systems against the security controls and even an experimental natural language processing utility for proofreading security narratives.
```

Comment:
s/also//

Maybe the link text should be "open source repository".

nit: "readable" (typo)

s/This also/this repository also

controls. An experimental natural language processing (NLP) utility is also included, for proofreading security narratives.

Reply:
Will update